app/src/main/java/fi/bitrite/android/ws/ui/listadapter/NavigationListAdapter.java

```diff
@@ -48,6 +51,8 @@ private NavigationListAdapter(Context context, List<Item> items,
                                   Observable<NavigationItem> onClickSubject) {
         super(context, R.layout.item_navigation, items);
         mOnClickSubject = onClickSubject;
+
+        locale = Tools.getLocale(getContext());
```

Comment:
Should `locale` really be static if it depends on the context and is set every time we create a `NavigationListAdapter`?